### PR TITLE
SCRUM-64: Add password change feature

### PR DESCRIPTION
**Pull request by Mohamed-Hammady**
made the controller, and the service for the password change feature

### CODE REVIEW

File: backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -3,8 +3,8 @@ spring.application.name=backend
 spring.datasource.url=jdbc:mysql://localhost:3306/skyrise
 
 # These properties specify the data used for authentication and authorization for the database
-spring.datasource.username= skyrise_admin
-spring.datasource.password= skyrise_app_swe26
```
Comment (Rawan-M-Said) [inline backend/src/main/resources/application.properties:7]: Revert the change here also for the user name and password

Comment (Mohamed-Hammady) [reply backend/src/main/resources/application.properties:7]: okay
